docs/extras/updating.md

```diff
@@ -79,40 +79,33 @@ If you keep your emuMMC offline, you will have to use a gamecard to update your
 
 !!!tip ""
     - The latest release of <a href="https://github.com/suchmememanyskill/TegraExplorer/releases" target="_blank">TegraExplorer</a>
-    - The latest release of <a href="https://switchtools.sshnuke.net" target="_blank">ChoiDujourNX</a>
     - The latest release of <a href="https://github.com/Atmosphere-NX/Atmosphere/releases" target="_blank">Atmosphere</a>
 
-!!!danger ""
-    ChoiDujourNX is **not** the same as ChoiDujour. Please make sure you download ChoiDujour**NX**!
-
 ### Preparing your SD card
 
 1. Insert your microSD card into your computer.
 2. Make a folder inside your `switch` folder called `system updates`.
 3. Download `TegraExplorer.bin` and place it somewhere on your PC (A good idea is to put it where you keep all your switch payloads like Hekate).
-3. Copy `ChoiDujourNX.nro` to `/switch/system updates` from the `ChoiDujourNX.zip` file.
 4. Update Atmosphere and Hekate by using the above guides.
 5. If you haven't already, update the sysMMC to the latest firmware.
 
 ### Dumping your sysMMC firmware
 
 1. Make sure your sysMMC is up to date. If your sysMMC is not up-to-date, update it through the System Settings.
 2. Inject `TegraExplorer.bin` using TegraRCMGUI (Like you would with Hekate).
-3. Using the volume and power buttons, select `Tools` and then `Dump Firmware`.
+3. Using the volume and power buttons, select `Tools`, then `Dump Firmware` and for the format type choose `Daybreak`.
 4. Wait about 1-2 minutes for the tool to dump your firmware.
 5. When the tool finishes, press any volume button.
-6. Select `Exit` and then `Reboot to Hekate`.
```

Comment:
Why was this swapped?

Reply:
Nevermind, selecting `Reboot to Hekate` reboots to `/bootloader/update.bin` and selecting `Reboot to Atmosphere` reboots to `/Atmosphere/reboot_payload.bin`. The former could encounter issues as not many people even have that file; and the latter would open the emuMMC anyways even if hekate wasn't their reboot payload.